Presentation.Web/app/models/it-contract/it-contract.ts

```diff
@@ -59,6 +59,8 @@
         Children: Array<IItContract>;
         /** Gets or sets the chosen agreement elements. */
         AgreementElements: Array<IAgreementElement>;
+        CriticalityId: number,
+        CriticalityType: ICriticalityType,
```

Comment:
cleanup after the DTO in the backend has been changed

Reply:
This one is the OData model for the overview so if you are using the properties then keep these,